### ISSUE TITLE
support Microsoft Office Publisher

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4653]
**Reported by blindbhavya on 2014-11-30 12:17**
Hi,
NVDA does not support Microsoft Office Publisher 2007 well enough to be usable, thats the very first impressions I have got, because of the following inaccessible parts:
Open a blank Microsoft Office Publisher 2007 file, and Tab around. In some palces, NVDA won't read anything on a Tab.
JAWS, does read those though, which are the Close Catalog button, Geting Started button, and some other things.
Please make nVDA support Microsoft Office publisher as well, especially since in the last few versions you guys have greatly enhanced accessibility for other Microsoft Office applications.
By the way, there may be other parts of this program that mya be inaccessible, please someone investigate.

**Comment by Adriani90** [comment]
@bhaviahshah are you still having this issue?

**Comment by DrSooom** [comment]
The elements on a page are still fully inaccessible with NVDA 2019.2.1 and Microsoft Publisher 2010 on Win7x64. And the Mouse Tracking only reads the complete line through the total length of a page, even if their content is part of separate text fields, which aren't linked together.

**Comment by ollcayy** [comment]
I have a project in my company that requires the use of publishers on office 365.
I need to create the text content of a project along with its baselines. The progress of my other teammates depends entirely on this. When it is finished, the project brochure that the agency has been waiting for from us will have appeared. Reading the text content on the page, however, NVDA with the Keyboard does not allow navigating the content, does not read the texts or is not aware of the changes.

**Comment by Adriani90** [comment]
cc: @Qchristensen, @feerrenrut maybe you can have a look on this, if the time permits. In corporate environments publisher is quite common software. Were there any conversations with Microsoft on this program sofar?

**Comment by ollcayy** [comment]
No, I state that the problem can be solved by screen readers.

 

Olcay Aşcı

 <https://certification.nvaccess.org/> NVDA Uzmanlık Sertifikası (NVDA Certified Expert)

Front-end Developer (Arayüz geliştirici)  <https://www.vodafone.com.tr/index.php> Vodafone TR Dijital Varlıklar Müdürlüğü

 <http://eniyideneyim.com/> En İyi Deneyim projesi yöneticisi ve sahibi

Takip edin:

 <https://twitter.com/eniyideneyim> Twitter'de en iyi deneyim  <https://twitter.com/olcayashchi> Twitter'da Olcay Aşcı

 <https://www.linkedin.com/in/olcay-a%C5%9F%C3%A7%C4%B1-b0a619124/> LinkedIn Olcay Aşcı  <https://www.facebook.com/profile.php?id=100000061230801> FaceBook Olcay Aşcı

**Comment by Adriani90** [comment]
Please don‘t make such controversial statements without details. Do you have some technical details prooving that Publisher exposes this properly to screen readers? If not, then please let‘s continue the discussion preferably with meaningful comments. Thanks.

Von meinem iPhone gesendet

**Comment by Adriani90** [comment]
For what is worth the MS Publisher will reach end of life in October 2026 and afterwards it will not be included in MS 365 anymore.

**Comment by seanbudd** [comment]
Closing as EOL is October 2026
